go/plugin/coredns/pfdns/pfdns.go

```diff
@@ -118,34 +112,13 @@ func (pf *pfdns) HasSecurityEvents(ctx context.Context, mac string) bool {
 }
 
 func (pf *pfdns) RefreshPfconfig(ctx context.Context) {
-	id, err := pfconfigdriver.PfconfigPool.ReadLock(ctx)
-	if err == nil {
-		defer pfconfigdriver.PfconfigPool.ReadUnlock(ctx, id)
-
-		// We launch the refresh job once, the first time a request comes in
-		// This ensures that the pool will run with a context that represents a request (log level for instance)
-		pf.refreshLauncher.Do(func() {
-			ctx := ctx
-			go func(ctx context.Context) {
-				for {
-					pfconfigdriver.PfconfigPool.Refresh(ctx)
-					err = pf.detectVIP(ctx)
-					if err != nil {
-						log.LoggerWContext(ctx).Error(err.Error())
-					}
-					time.Sleep(1 * time.Second)
-				}
-			}(ctx)
-		})
-	} else {
-		panic("Unable to obtain pfconfigpool lock in pfdns middleware")
-	}
+	// pf.pfdnsRefreshableConfig = pfconfigdriver.GetRefresh(ctx, "pfdnsRefreshableConfig").(*pfdnsRefreshableConfig)
```

Comment:
refresh not needed?

Reply:
The refresh is done elsewhere in the code.